### DISCUSSION TITLE
2023-08-23 Weekly Meeting 121 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#413] [Weekly Meetings]
Weekly Meeting 121 is happening on Wednesday, August 23, 2023 from 9:00 AM PDT (16:00 UTC) - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

Agenda

UPDATES
0. Lots of agenda items, start with asking what is urgent and dealing with that first.
1. Recap from [last week's meeting](https://github.com/omigroup/omigroup/discussions/412). Notes? Recording? Fireflies.ai?
2. Media group: 
2.1 We need to get in touch with a virtual burner to have a campfire event, chat about lessons from their 2020 multiverse initiative, possible colaboration. [Media meeting notes](https://github.com/omigroup/media/discussions/66#discussioncomment-6732337) Does anyone know anyone?
3. Date range for [Metaverse festival]((https://hackmd.io/@indiebio/metaverse-float-parade)) - mid October 2024. What clashes to look out for?
4. XR showcase: 
4.1 Feedback from [M3 Demo Day](https://discord.com/channels/770382203782692945/830454514682167356/1143022786789507165) - jin
4.2 "Anybody know any other online communities bringing together XR devs from different projects to share their work via in-world events besides M3?" - via [jin comment](https://discord.com/channels/770382203782692945/830454514682167356/1143246496360321084). Is this something we can host as OMI?
5. Funding from metaverse festival, how to get money in, and what to do with it if we have it - see [jin comment](https://discord.com/channels/770382203782692945/830454514682167356/1142453952949006456) 
6. glTF working group
7. MSF 
8. Other working group updates?
9. Call for nominations and volunteers for the next election. Is this for W3C or internal for OMI? Is there a difference?
9.1 [indiebio nomination](https://github.com/omigroup/omigroup/discussions/410#discussioncomment-6683620)
10. AOB

**Comment by indiebio** [comment]
meetup in https://upstreet.ai/m/#0,0?

**Comment by indiebio** [comment]
Burners: ask Evo for contacts.

**Comment by indiebio** [comment]
4.2 "Anybody know any other online communities bringing together XR devs from different projects to share their work via in-world events besides M3?" [ in virtual]  - there are ecosystems, like edtech communities, - @avatarjoy to give links please. 
OpenSIM conference (not VR capable / not as VR focused but there is a VR ...browser thing) completely FOSS. Host your own server etc. 
MalBurns: with focus on interop not great, slow ... 

WebXR there are possibilities, Hubs ... web-based may be good, can easier jump between worlds / tabs...? May be better for interop.
There, Mozilla Hubs have hosted conferences in virtual.
M3 and webXR monthly meetups are active.

Metatraversal series of events also. @evo works with them. @KEYFRAME also involved there, can speak to potential collaborations. more info on MetaTr@versal https://www.youtube.com/channel/UCBRnEdbmKvanvN8rtfnzApw

**Comment by indiebio** [comment]
Could someone with Google access please add the media group meeting to the OMI calendar?

**Reply by indiebio** [reply]
Need to get access, need to get new people (elections)

**Comment by indiebio** [comment]
5. Funding, figure it out later, for now do peer-to-peer

**Comment by aaronfranke** [comment]
glTF working group: We are working on syncing with Microsoft's physics https://github.com/omigroup/gltf-extensions/pull/185. Lyuma suggested to wait until Microsoft clarifies the trigger grouping behavior. iFire suggested we continue independently.

**Comment by indiebio** [comment]
MSF: end of November do a retrospective about why meetings are low energy,low engagement. Have many many members, but very few shop up. Silo issues. Low collaboration in-between. 
wrt virtual conference, virtual avatars group has momentum, wearables group good traction... 
Needs focus? A clear objective? Needs something actionable to take away. "WIIFM?" What's in it for people to contribute?
Better agenda items. Not well organised. 
A clear vision? What is the answer to "So What?"
Not much engagement outside of meetings.
@jin [comments on Discord](https://discord.com/channels/770382203782692945/830454514682167356/1143947268785979433)

**Comment by indiebio** [comment]
![image](https://github.com/omigroup/omigroup/assets/23474368/4b093f2f-5b31-4c37-b690-fb50c9e64854)
via @madjin

**Reply by indiebio** [reply]
from anonymous:
This is basically how the last few years have gone:
OMI: Hey, let's standardize the metaverse!
P1-OM: Hey, let's build the metaverse!
MSF (Facebook): A metaverse standards org open to anyone? That's dumb, we should make our own private org that only companies are invited to so we control everything.
IDEA / ITMF: 3 metaverse standards groups? Ridiculous! We need to develop one universal standard...
OMF (The Linux Foundation): 4 metaverse standards groups? Ridiculous! We need to develop one universal standard...
MPEG: 5 metaverse standards groups? Ridiculous! We need to develop one universal standard...
ITF (United Nations): 6 metaverse standards groups? Ridiculous! We need to develop one universal standard...
(and yeah I'm not joking the United Nations has a metaverse standards org)

**Reply by indiebio** [reply]
How active are OMI working groups?
Does anyone other than OMI work with Khronos?
USD is also a good thing to mention. but glTF is alsovery good and used/backed by many open source projects and groups. But interoperability between them is possible and supported, they're not necessarily in conflict.

**Comment by indiebio** [comment]
9. How has nominations worked in the past? Historically no change. The champions are self-selected, which is separate from W3C. Is there an improved way to do this?
Best to discuss this in the Champions meeting.
